packet/flatcar-linux/kubernetes/variables.tf

```diff
@@ -49,11 +49,6 @@ variable "worker_nodes_hostnames" {
   description = "List of hostname of packet_device resources"
 }
 
-variable "worker_nodes_public_ipv4s" {
-  type        = "list"
-  description = "List of public IPv4 of packet_device resources"
-}
-
```

Comment:
Doing this, now that is not needed, is what I was saying that breaks the API of the module. As current users will break. https://github.com/kinvolk/lokomotive-kubernetes/pull/13#discussion_r292844280

As I said, not sure if we want to deprecate/terraform gives us a nice way to show a warning for some time or, as the platform support is in Beta we can do these kind of changes.

At this stage I'd lean towards removing this without deprecation, specially since there is no release yet, but is something we should all agree on, I think

Reply:
I've updated the comments to remove this variable so new deployments should not have it.

As for breaking existing stuff, as long as people use `ref=<sha1>` in TF module sources nothing should break. If people don't use a ref they should expect breaks. We may want to emphasize this fact in the docs.